lib/code_corps/analytics/segment_traits_builder.ex

```diff
@@ -6,7 +6,7 @@ defmodule CodeCorps.Analytics.SegmentTraitsBuilder do
   @spec build(struct) :: map
   def build(record), do: traits(record)
 
-  defp traits(%CodeCorps.User{} = user) do
+  defp traits(comment = %CodeCorps.User{}) do
```

Comment:
`user`

Also, why does the variable name go first here, while it goes second in `deep_resolve` above? Was the credo warning different?

Reply:
Whoops! Good catch.

Ah, y'know I think credo wanted consistency around parameters names. The check in particular, `ParameterPatternMatching` I turned off after fixing a couple, because there were too many instances of it.

Reply:
Hopefully the upcoming Elixir built-in code formatting fixes some of this, too!